### PR TITLE
Move daal4py examples to daal4py folder

### PR DESCRIPTION
**Pull request by PetrovKP**
# Description
Please include a summary of the change. For large or complex changes please include enough information to introduce your change and explain motivation for it.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

**Comment by PetrovKP** [comment]
/intelcI: run
set daal-infra=move-repo

**Comment by PetrovKP** [comment]
/intelci: run
set daal-infra=move-repo

**Comment by PetrovKP** [comment]
/intelci: restart
set infra_branch=move-repo

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 #===============================================================================
 
-{% set version = '2021.1' %}
-{% set buildnumber = 4 %}
+{% set version = '2021.2' %}
```
Comment (napetrov) [inline conda-recipe/meta.yaml:17]: why not 2021.3 ? It's current one. More over assuming created branches master should be 2021.4

**Review by michael-smirnov — APPROVED** [review]
LGTM
Please run private CI

**Review by napetrov — APPROVED** [review]

**Comment by PetrovKP** [comment]
/intelci: restart
set infra_branch=move-repo

**Pull request merged**